README.md

```diff
@@ -245,7 +247,9 @@ when initializing your instance of the SplunkRum API:
   Set/change the default polling interval for slow/frozen render detection.
   Default is 1000ms. Value must be positive. 
 - `disableSlowRenderingDetection()` :
-  Disable the detection of slow frame renders (default is enabled). 
+  Disable the detection of slow frame renders (default is enabled).
+- `enableDiskBuffering()` : 
+  Enables the storage-based buffering of telemetry.
```

Comment:
@jkwatson @breedx-splk Sounds like this might come in handy in certain scenarios. What are those?

Reply:
For example, when you have an app that may work offline for extended periods of time (like a GPS trail tracker)

Reply:
```suggestion
  Enables the storage-based buffering of telemetry. This setting is useful when instrumenting applications that might work offline for extended periods of time.
```

Reply:
Eventually we want this to be the default....but for now it's opt-in because it's new.